### PR TITLE
EDU-1532 Create star rating component

### PR DESCRIPTION
**Pull request by ahelmberger**
Closes https://educandu.atlassian.net/browse/EDU-1532

### CODE REVIEW

File: src/components/star-rating.less [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  grid-auto-flow: column;
+}
+
+.StarRating-value {
+  padding-right: 10px;
+}
+
+.StarRating-rating {
+  .ant-rate-star {
+    margin-inline-end: 0 !important;
```
Comment (ahelmberger) [inline src/components/star-rating.less:14]: Unfortunately, the AntD rate component has everything hard-coded (including the star colors), so we cannot easily customize. But IMO the stars had too much space in between by default (8px), so I removed the space here forcefully.

File: src/components/star-rating.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+starRating:
+  totalCountText:
+    en: "{totalCount, plural, =0 {no star ratings yet} =1 {# star rating} other {# star ratings}}"
+    de: "{totalCount, plural, =0 {noch keine Sternebewertungen} =1 {# Sternebewertung} other {# Sternebewertungen}}"
```
Comment (adrianaluchian) [inline src/components/star-rating.yml:4]: I would suggest dropping the "star" from "star rating", as rating is the feature and star is just the representation. It sounds a bit much to me. Maybe even dropping "yet" / "noch".

Comment (adrianaluchian) [reply src/components/star-rating.yml:4]: After seeing it, I'm even wondering if maybe we should just put the count in parenthesis and have a tooltip saying something like "Number of ratings" or "X users have given a rating"? What do you think? 🤔

Comment (ahelmberger) [reply src/components/star-rating.yml]: I have reworked the styling based on our discussion.

**Review by adrianaluchian — APPROVED** [review]

**Pull request merged**